graylog2-web-interface/src/views/components/messagelist/MessageFields/MessageFieldsViewModeList.tsx

```diff
@@ -15,33 +15,38 @@
  * <http://www.mongodb.com/licensing/server-side-public-license>.
  */
 
-import React from 'react';
+import React, { useContext } from 'react';
 import styled, { css } from 'styled-components';
 
 import type { MessageFieldsListProps } from 'views/components/messagelist/MessageFields/types';
 import MessageField from 'views/components/messagelist/MessageField';
-import { MessageDetailsDL } from 'views/components/messagelist/MessageFields/MessageFields';
+import { MessageDetailsDefinitionList } from 'components/common';
+import MessageFavoriteFieldsContext from 'views/components/contexts/MessageFavoriteFieldsContext';
 
-const Container = styled.div(
+const MessageDetailsDL = styled(MessageDetailsDefinitionList)(
   ({ theme }) => css`
-    margin-left: ${theme.spacings.sm};
+    color: ${theme.colors.text.primary};
+
+    dd {
+      &:not(:last-child) {
+        border-bottom: 1px solid ${theme.colors.table.row.divider};
+      }
+    }
   `,
 );
 
-const MessageFieldsViewModeList = ({ fields, message, isFavorite }: MessageFieldsListProps) => {
+const MessageFieldsViewModeList = ({ fields }: MessageFieldsListProps) => {
```

Comment:
The naming of the component is not really comprehensible to me. Can we use something simpler, like `MessageFieldsList`?

Reply:
We have also MessageFieldsEditModeLists. They are different. The naming is to make clear what this list are about